django/santropolFeast/order/models.py

```diff
@@ -147,8 +146,7 @@ def create_orders_on_defaults(creation_date, delivery_date, clients):
             if free_side_dish_qty == 0:
                 continue  # No meal for client on this day
             try:
-                order = Order.objects.get(
-                    client=client, delivery_date=delivery_date)
+                Order.objects.get(client=client, delivery_date=delivery_date)
                 # order already created, skip order items creation
                 # (if want to replace, must be deleted first)
                 continue
```

Comment:
> @erozqba Thanks for your great contributions, For this one, in file django/santropolFeast/order/models.py in line 150 you changed
> order = Order.objects.get(client=client, delivery_date=delivery_date)
> for Order.objects.get(client=client, delivery_date=delivery_date)
> However, the order = is necessary because when the try does not raise an exception, in line 188 we will need order to have been previously defined
> oi = Order_item(order=order, ...
> 

Hi @lamontfr, thanks a lot for your revision, I don't find your comment here in Github, so I have pasted the content of the mail to have the proper context for my question. I change the line 150 because all this code is inside a _**for**_ loop and in line 152, inside the **_try_** there is a _**continue**_, so if an exception is not raised the execution of the code will follow with the next iteration of the loop and will only get to the line 186 if an exception is raised. Have I missed something there?